src/components/AddNewBook.js

```diff
@@ -1,27 +1,45 @@
 import React, { useState } from 'react';
-import { v4 as uuidv4 } from 'uuid';
 import { useDispatch } from 'react-redux';
-import { addBook } from '../redux/Books/books';
+import { v4 as uuidv4 } from 'uuid';
+import { postBook } from '../redux/Books/books';
 
 const AddNewBook = () => {
   const [inputValues, setInputValues] = useState({
-    booktitle: '',
+    title: '',
     author: '',
+    id: '',
     category: '',
   });
   const dispatch = useDispatch();
+  const [errorMsg, setError] = useState('');
 
   const submitBookToStore = (e) => {
     e.preventDefault();
     const id = uuidv4();
-    const { booktitle, author, category } = inputValues;
+    const { title, author, category } = inputValues;
     const newBook = {
-      booktitle,
-      author,
       id,
+      title,
+      author,
       category,
     };
-    dispatch(addBook(newBook));
+
+    if (newBook.title.trim().length === 0) {
+      setError('Please add Book title to submit...');
+      setInputValues(newBook);
+    } else if (newBook.category === '') {
```

Comment:
- [ OPTIONAL] Please add also the validation error message to the author field as you did with the title 😉 👍🏻 
- eg: Please add Book author to submit...